test/parallel/xyk-pallet.multiswap.errors.test.ts

```diff
@@ -154,20 +148,22 @@ describe("Multiswap - error cases: pool status & gasless integration", () => {
     );
     let exception = false;
     await expect(
-      multiSwapSell(testUser1, tokenIds, threshold.addn(10)).catch((reason) => {
-        exception = true;
-        throw new Error(reason.data);
-      }),
-    ).rejects.toThrow(feeLockErrors.FeeLockingFail);
+      multiSwapSellMarket(testUser1, tokenIds, threshold.addn(10)).catch(
+        (reason) => {
+          exception = true;
+          throw new Error(reason.data);
+        },
+      ),
+    ).rejects.toThrow(feeLockErrors.AccountBalanceFail);
```

Comment:
feeLockErrors.FeeLockingFail !== feeLockErrors.AccountBalanceFail
is this test skipped, or the error changed on this specific test?

Reply:
I suppose that before that we can pay fees in pool token, but right now use must pay fees in GASP, which we don't mint for this account